src/Dockerfile

```diff
@@ -3,7 +3,46 @@ FROM python:3.10.8
 RUN useradd --create-home --shell /bin/bash bananabot
 
 USER bananabot
```

Comment:
Notes for myself:
Don't forget to test this Dockerfile works prior to merging

Reply:
Having issues currently with the server shutting itself down when any `POST` request is sent after BananaBot has been toggled on. Still in the process of fixing it

Reply:
The Docker image / file now functions as expected. The issues I was having appear to have been caused by an issue documented here: https://github.com/tiangolo/fastapi/issues/1487
Due to this I made the decision to switch from using Python Multiprocessing to using Python Threads instead. This avoids the issue and seems to run smoother than using Processes too 😄